src/sentry/templates/sentry/integrations/generic-error.html

```diff
@@ -0,0 +1,14 @@
+{% extends "sentry/bases/modal.html" %}
```

Comment:
Is this a new view? Do you have a screenshot of it?

Reply:
No, it's an exact duplicate of `slack/link-team-error.html`, which was never specific to Slack nor to team-linking. I just copied it to a new path for clarity.

Deleting the file from the old path is another todo.